app/views/admin/loans/index.html.slim

```diff
@@ -0,0 +1,3 @@
+- content_for(:title, content='Loans')
```

Comment:
I don't think you need the `content=` bit here. I could be wrong though. Maybe it's used further down?